ui/components/FlipCard/FlipCard.style.js

```diff
@@ -0,0 +1,31 @@
+import { styled } from "@mui/material/styles";
+
+export const Container=styled('div')(({theme})=>({
+  position:"relative",
+  height:"100%",
+  borderRadius : theme.spacing(1),
+}))
+
+export const InnerCard=styled('div')(({flipped,theme})=>({
```

Comment:
Use `Paper` here , and remove the background, position, border radius and change the name of InnerCard to CardWrapper